math/README.md

```diff
@@ -0,0 +1,5 @@
+## Math Concepts
+
+| Paper | Notes | Start Date | End Date |
+| -------- | -------- | -------- | -------- | -------- |
```

Comment:
@SharathRaparthy there's something wrong with the rendering of the table. one column is extra I think.